morphia/src/main/java/org/mongodb/morphia/Morphia.java

```diff
@@ -160,19 +165,23 @@ public boolean isMapped(final Class entityClass) {
     }
 
     /**
-     * @return true if Morphia should use bulk write operations.  Only useful with MongoDB 2.6+.
+     * @return false.  Setting this value has no value functionally or performance-wise.
+     * @see <a href="https://github.com/mongodb/morphia/issues/1052">Issue #1052</a>
      */
     public boolean isUseBulkWriteOperations() {
-        return useBulkWriteOperations;
+        return false;
     }
 
     /**
      * Configures Morphia to use bulk writes.  Only useful with MongoDB 2.6+.
      *
      * @param useBulkWriteOperations true if Morphia should use bulk writes
+     * @see <a href="https://github.com/mongodb/morphia/issues/1052">Issue #1052</a>
+     * @deprecated Setting this value has no value functionally or performance-wise.
      */
+    @Deprecated
     public void setUseBulkWriteOperations(final boolean useBulkWriteOperations) {
-        this.useBulkWriteOperations = useBulkWriteOperations;
+        LOG.warning("Setting useBulkWriteOperations has no value functionally or performance-wise.");
```

Comment:
Visibility mainly.  An ongoing warning that things have changed.  Figured it was the polite/correct thing to do.